### DISCUSSION TITLE
Does winget support installing programs onto another drive?

### DISCUSSION THREAD

**Discussion by OakBricks-old** [microsoft/winget-cli#1281] [Q&A] [answered]
My C: drive is currently limited on space (about 9GB free) and I was wondering if winget supports installing programs onto another drive (such as D: or E:).

**Comment by ImJoakim** [comment, marked as answer]
There is currently a setting/option (if supported/honored by the installer) which is `winget install <package> --location <path>`

Example: `winget install vscode --location "M:\Program Testing\VSCode"`

**Reply by OakBricks-old** [reply]
ok thanks!
